content/applications/hr/time_off/allocations.rst

```diff
@@ -0,0 +1,118 @@
+
+  Multiple selections can be made for either :guilabel:`Employees` or :guilabel:`Employee Tag`.
+
+  Only one selection can be made for the :guilabel:`Company` or :guilabel:`Department`.
+- :guilabel:`Add a reason...`: if any description or note is necessary to explain the time off
+  allocation, enter it in this field at the bottom of the form.
+
+.. image:: allocations/new-allocation.png
+   :align: center
+   :alt: A new allocation form with all the fields filled out for the annual two week vacation
+         granted to all employees.
+
+.. _time_off/request-allocation:
+
+Request allocation
+==================
+
+If an employee has used all their time off, or will run out of time off, they can request an
+allocation for additional time. Allocations can be requested in one of two ways, either from the
+:ref:`Dashboard <time_off/dashboard>` or the :ref:`My Allocations <time_off/my-allocations>` view.
+
+To create a new allocation request, click either the :guilabel:`New Allocation Request` button on
+the main *Time Off* dashboard, or the :guilabel:`New` button in the :guilabel:`My Allocations` list
+view. Both buttons open a new allocation request form.
+
+.. note::
+   Both options open a new allocation request form, but when requested from the
+   :guilabel:`Dashboard`, the form appears in a pop-up window. When requested from the
+   :guilabel:`My Allocations` list view, the screen navigates to a new allocation request page,
+   instead.
+
+Enter the following information on the new allocation request form:
+
+- :guilabel:`Time Off Type`: select the type of time off being requested for the allocation from the
+  drop-down menu. After a selection is made, the title updates with the time off type.
+- :guilabel:`Validity Period`: using the calendar selector, select a start and end date the
+  allocation is valid for. By default, the current date populates the first field, and :guilabel:`No
+  Limit` populates the second field. Adjust both of these dates, if desired.
+- :guilabel:`Allocation`: enter the amount of time being requested in this field. The format is
+  presented in either :guilabel:`Days` or :guilabel:`Hours`, depending on how the :guilabel:`Time
+  Off Type` is configured. Once this field is populated, the name of the allocation request is
+  updated to include the amount of time being requested.
+- :guilabel:`Add a reason...`: enter a description for the allocation request. This should include
+  any details that approvers may need to approve the request.
+
+If the request was created from the :guilabel:`Dashboard`, click the :guilabel:`Save & Close` button
+on the :guilabel:`New Allocation` pop-up window to save the information and submit the request.
+
+If the form was completed from the :guilabel:`My Allocations` list view, the information is
+automatically saved as it is entered. However, the form can be saved manually at any time by
+clicking the :icon:`fa-cloud-upload ` :guilabel:`(cloud upload)` icon.
```

Comment:
This is where your [build error](https://runbot.odoo.com/runbot/build/65280575) is coming from :)
```suggestion
clicking the :icon:`fa-cloud-upload` :guilabel:`(cloud upload)` icon.
```